config/ad_config.py

```diff
@@ -0,0 +1,595 @@
+#!/usr/bin/python3
+# coding=utf-8
+"""
+Automate demultiplex configuration. Contains the following settings:
+- General settings used across modules
+- Demultiplexing script-specific settings
+- Setoff workflows script-specific settings
+"""
+import os
+import datetime
+from pygit2 import Repository
+from pathlib import Path
+from .log_msgs_config import LOG_MSGS
+from .panel_config import TOOLS_PROJECT, MASKED_REFERENCE, PanelConfig
+
```

Comment:
There are hardcoded paths (e.g., "/media/data3/share" for RUNFOLDERS) and potential secrets (e.g., email addresses). For better security and flexibility, consider putting these settings into environment variables or a secure configuration service.

Reply:
I agree, I will make a GitHub issue for this for the next update